source/_docs/configuration/templating.markdown

```diff
@@ -518,7 +518,8 @@ Some of these functions can also be used in a [filter](https://jinja.palletsproj
 - Test `string is match(find, ignorecase=False)` will match the find expression at the beginning of the string using regex.
 - Test `string is search(find, ignorecase=True)` will match the find expression anywhere in the string using regex.
 - Filter `string|regex_replace(find='', replace='', ignorecase=False)` will replace the find expression with the replace string using regex.
-- Filter `string|regex_findall_index(find='', index=0, ignorecase=False)` will find all regex matches of find in string and return the match at index (findall returns an array of matches).
+- Filter `string|regex_findall(find='', ignorecase=False)` will find all regex matches of the find expression in string and return the array of matches.
+- Filter `string|regex_findall_index(find='', index=0, ignorecase=False)` will do the same as `regex_findall` and return the match at index.
```

Comment:
```suggestion
- Filter `value | regex_findall(find='', ignorecase=False)` will find all regex matches of the find expression in `value` and return the array of matches.
- Filter `value | regex_findall_index(find='', index=0, ignorecase=False)` will do the same as `regex_findall` and return the match at index.
```
1) It does not have to be a string,  `{{ 123  | regex_findall(find='', ignorecase=False) }}` is valid as well
2) Filter should have spacing around them <https://developers.home-assistant.io/docs/documenting/yaml-style-guide#filters>